pom.xml

```diff
@@ -263,6 +266,25 @@
                 <artifactId>guava</artifactId>
                 <version>${guava.version}</version>
             </dependency>
+            <!-- we define aws-java-sdk azure-identity and azure-storage-blob
+            versions above, to match the versions in ce-kafka -->
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
+                <version>${aws-java-sdk.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.azure</groupId>
+                <artifactId>azure-identity</artifactId>
+                <version>${azure-identity.version}</version>
```

Comment:
Should we also place a scope <scope>import</scope> for azure?

Reply:
no, this is regular dependency, only bom typed dependencies should be in scope import

Reply:
see line 375/397 for netty-bom